MMM-COVID19.js

```diff
@@ -87,39 +88,51 @@ Module.register("MMM-COVID19", {
 
     // header row
     var headerRow = document.createElement("tr"),
-        headerconfirmedCell = document.createElement("td"),
+        headerConfirmedCell = document.createElement("td"),
+        headerCasesPerMCell = document.createElement("td"),
         headerNewConfirmedCell = document.createElement("td"),
         headerCountryNameCell = document.createElement("td"),
-        headerrecoveredCell = document.createElement("td"),
-        headerdeathsCell = document.createElement("td"),
+        headerRecoveredCell = document.createElement("td"),
+        headerDeathsCell = document.createElement("td"),
         headerNewDeathsCell = document.createElement("td"),
-        headeractiveCell = document.createElement("td")
+        headerSeriousCell = document.createElement("td"),
+        headerActiveCell = document.createElement("td");
 
     headerCountryNameCell.innerHTML = ''
-    headerconfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
-    headerconfirmedCell.innerHTML = this.translate('Confirmed')
+    headerConfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
+    headerConfirmedCell.innerHTML = this.translate('Confirmed')
+    headerCasesPerMCell.className = 'number per M ' + this.config.headerRowClass
+    headerCasesPerMCell.innerHTML = this.translate('Cases per M')
     headerNewConfirmedCell.className = 'number confirmed ' + this.config.headerRowClass
     headerNewConfirmedCell.innerHTML = this.translate('New Cases')
-    headerdeathsCell.className = 'number deaths ' + this.config.headerRowClass
-    headerdeathsCell.innerHTML = this.translate('Deaths')
+    headerDeathsCell.className = 'number deaths ' + this.config.headerRowClass
+    headerDeathsCell.innerHTML = this.translate('Deaths')
     headerNewDeathsCell.className = 'number deaths ' + this.config.headerRowClass
     headerNewDeathsCell.innerHTML = this.translate('New Deaths')
-    headerrecoveredCell.className = 'number recovered ' + this.config.headerRowClass
-    headerrecoveredCell.innerHTML = this.translate('Recovered')
-    headeractiveCell.className = 'number active ' + this.config.headerRowClass
-    headeractiveCell.innerHTML = this.translate('Active')
+    headerSeriousCell.className = 'number deaths ' + this.config.headerRowClass
```

Comment:
@testdruif wrong className used here. Don't use the same color as Deaths. If you want to highlight it someway, please add a new class called `serious` and add the proper entry at the .css file of the module